server/plugin/utils.go

```diff
@@ -45,10 +45,12 @@ func getIssuesSearchQuery(org, searchTerm string) string {
 	return fmt.Sprintf(query, orgField, searchTerm)
 }
 
-func buildSearchQuery(query, username, org string) string {
+func buildSearchQuery(query, username string, orgs []string) string {
 	orgField := ""
-	if len(org) != 0 {
-		orgField = fmt.Sprintf("org:%v", org)
+	for _, org := range orgs {
+		if len(org) != 0 {
```

Comment:
Can `org` be `""`?

Reply:
Yes, if a user enters `orgA,orgB,` as values in the system console, the last value after splitting the string will be an empty string.

Reply:
Should we guard for that case in `getOrganizations`?

Reply:
> Should we guard for that case in getOrganizations?

@hanzei Can you please elaborate more on this comment? We have already added the guard to the `getOrganizations` function. Do you mean to remove it from here and apply the condition separately, wherever it is required?

Reply:
The slice returned from `getOrganizations` can never contain an empty string, right? Do we need the additional check here?

Reply:
@hanzei There was an additional check present in the `lhs_request.go` file, which I have removed now. Now, we only have this check present in the `getOrganizations` function.